setup.cfg.jinja

```diff
@@ -1,15 +1,3 @@
-[options]
-package_dir =
-    = src
-packages = find:
-include_package_data = True
-
-[options.packages.find]
-where = src
-
-[options.package_data]
-{{projectname}} = py.typed
-
```

Comment:
We don't need any of this, just by bumping the versions in pyproject.toml? No other changes needed?

Reply:
Yes. It now recognizes (probably has for a long time) the src-layout that we use. And like I said, data files are now included by default if they are in the package tree and tracked by a VCS.